src/features/embalm/stepContent/components/SuccessfulCreateSarcophagus.tsx

```diff
@@ -12,87 +15,168 @@ export function SuccessfulCreateSarcophagus({
+        mb={6}
+      />
+
+      {/*
+       * DETAILS BOX AND CONTENT
+       */}
       <Flex
-        justifyContent="space-between"
-        wrap="wrap"
+        width={700}
+        direction="column"
       >
-        <Text
-          width="40%"
-          fontSize="sm"
-          mt={4}
-        >
-          Sarcophagus TX
-        </Text>
-        <Text
-          width="60%"
-          fontSize="sm"
-          textAlign="right"
-          mt={4}
-        >
-          {successSarcophagusTxId}
-        </Text>
-        <Text
-          width="40%"
-          fontSize="sm"
-          mt={4}
-        >
-          Arweave File TX
-        </Text>
-        <Text
-          width="60%"
-          fontSize="sm"
-          textAlign="right"
-          mt={4}
-        >
-          {successSarcophagusPayloadTxId}
-        </Text>
-        <Text
-          width="40%"
-          fontSize="sm"
-          mt={4}
+        <Heading
+          fontSize="md"
+          py="12px"
+          textAlign="center"
+          bgGradient="linear-gradient(180deg, rgba(255, 255, 255, 0.045) 0%, rgba(255, 255, 255, 0.09) 100%)"
+          border={boxBorder}
+          borderColor="brand.300"
+          borderBottom="none"
+          borderRadius="4px 4px 0px 0px"
         >
-          Arweave Archaeologist TX
-        </Text>
-        <Text
-          width="60%"
-          fontSize="sm"
-          textAlign="right"
-          mt={4}
+          Transaction Details
+        </Heading>
+
+        {/*
+         * CONTENT
+         */}
+        <Flex
+          direction="column"
+          border={boxBorder}
+          borderColor="brand.200"
+          borderTop="none"
         >
-          {successEncryptedShardsTxId}
-        </Text>
+          <Flex
+            justifyContent="space-between"
+            wrap="wrap"
+            px="18px"
+          >
+            {/*
+             * ARWEAVE SHARDS TRANSACTION ID
+             */}
+            <Text
+              width="40%"
+              fontSize="sm"
+              mt={4}
+            >
+              {/* TODO: This label is weird, almost meaningless */}
+              Archaeologist Arweave ID
+            </Text>
+            {arweaveTxId(successEncryptedShardsTxId)}
+            {copyIcon(hasCopiedShardsId, copyShardsId)}
+
+            {/*
+             * ARWEAVE PAYLOAD TRANSACTION ID
+             */}
+            <Text
+              width="40%"
+              fontSize="sm"
+              mt={4}
+            >
+              Arweave File ID
+            </Text>
+            {arweaveTxId(successSarcophagusPayloadTxId)}
+            {copyIcon(hasCopiedFileId, copyFileId)}
+
+            {/*
+             * SARCO CREATION TX ETHERSCAN LINK
+             */}
+            <Text
+              width="40%"
+              fontSize="sm"
+              mt={4}
+            >
+              Sarcophagus Creation
+            </Text>
+            <Text
+              fontSize="10px"
+              color="brand.700"
```

Comment:
I think `color='brand.700'` can be done with `variant='secondary'` also. I guess I would tend away from adding `color='anything'` unless I really have too.